.github/dependabot.yml

```diff
@@ -0,0 +1,169 @@
+version: 2
+updates:
+  # Root
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
I think this will be quite noisy for a while, but then we can bump the `interval` and `open-pull-requests-limit` up